test-utils/test-runner/src/utils.rs

```diff
@@ -20,10 +20,15 @@ use futures::{Sink, SinkExt};
 use std::fmt;
 use std::io::Write;
 use log::LevelFilter;
+use sc_service::BasePath;
 
 /// Base db path gotten from env
-pub fn base_path() -> Option<String> {
-	std::env::var("DB_BASE_PATH").ok()
+pub fn base_path() -> BasePath {
+	if let Some(base) = std::env::var("DB_BASE_PATH").ok() {
+		BasePath::new(base)
+	} else {
+		BasePath::new_temp_dir().expect("couldn't create a temp dir")
```

Comment:
Isn't this like a random path?

Reply:
yeah, it is, if you don't supply an existing db path, it is assumed you want to test with a new db

Reply:
yeah okay I was wondering if loading a fresh DB would make any sense, but yeah I presume you could also start with a new DB and it should be fine.